### DISCUSSION TITLE
Chatting platform to replace Gitter

### DISCUSSION THREAD

**Discussion by martinfleis** [pysal/pysal#1291] [Polls]
Hi all, 
you are probably aware that our Gitter is not active any longer and that the last attempt to make it work miserably failed. The thing just does not work (technically) as it should. During the last call, we agreed that we will switch to three modes of discussion:

1. Internal mailing list (for stuff every maintainer and core team member needs to know)
2. GitHub Discussions (for stuff that needs persistence and longer discussion)
3. New chatting platform (for stuff that happens right now and is a bit more ephemeral in nature)

The first two are straightforward. The last one needs a decision on which platform will be used. We have limited the selection to Slack and Discord and everyone is invited to vote which one would you prefer. Feel free to add reasons to the discussion thread below.

We don't want to pay for either of them, so the free plans come with some limitations. The only one that may be relevant for us is that Slack gives us history only for 90 days while Discord seems to provide full history.

**Comment by jGaboardi** [comment]
The 90-day history for Slack is a dealbreaker in my opinion, even if Discord feels like being in a kids cartoon.

**Reply by jGaboardi** [reply]
... but.... if we are diligent about curating important topics after their introduction in the chat, then Slack would be my preferred choice.

**Reply by martinfleis** [reply]
Important stuff should ideally happen on GitHub but things like "here's a nice paper citing spaghetti" would be nice to retain when we need to write a funding proposal in a year.

**Comment by knaaptime** [comment]
ive never used discord, so i dont know what not to like 🤷‍♂️

**Reply by martinfleis** [reply]
That is quite a good point to be fair. Better to stick to apps people already use, I'd say.

**Comment by jGaboardi** [comment]
@TaylorOshan – seems to be default behavior to be invite only.

**Comment by jGaboardi** [comment]
@gegen07 – says discord works great

**Comment by jGaboardi** [comment]
Decided in Discord (for now) – @martinfleis set up server >> [see invite here](https://github.com/orgs/pysal/discussions/1278#discussioncomment-8015349)
